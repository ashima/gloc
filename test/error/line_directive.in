#line 50
#line 09
#line 02
#line 2 0xF
#error error at file 15, line 2
