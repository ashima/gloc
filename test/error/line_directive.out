File 0, line 50, col 6 - 8:
invalid octal constant "09"

File 0, line 9, col 6 - 8:
line control arguments must be specified in decimal

File 0, line 2, col 8 - 11:
line control arguments must be specified in decimal

